docs/applications/readarr.mdx

```diff
@@ -0,0 +1,91 @@
+---
+id: readarr
+title: Readarr
+sidebar_label: Readarr
+---
+
+Readarr is a ebook (and maybe eventually magazine/audiobook) collection manager for Usenet and BitTorrent users. It can monitor multiple RSS feeds for new books from your favorite authors and will grab, sort and rename them.
```

Comment:
```suggestion
Readarr is an ebook (and maybe eventually magazine/audiobook) collection manager for Usenet and BitTorrent users. It can monitor multiple sources (RSS, Torznab, etc.) for new books from your favorite authors and will grab, sort and rename them for you.
```

Reply:
RSS is the torznab or usenet, etc feed.

RSS is not a unique feed